WordPress/MediaEditor/MediaEditor.swift

```diff
@@ -53,17 +88,65 @@ public class MediaEditor: UINavigationController {
         present(capability: capabilityEntity, with: image)
     }
 
+    private func cancel() {
```

Comment:
Is this going to be called from anywhere else other than as the onCancel block for present?

Reply:
I don't think so. This might change in the future but this will be called only when the MediaEditor is dismissed.